agents/mongodb/internal/profiler/collector/collector_test.go

```diff
@@ -202,8 +203,12 @@ func createSession(dsn string, agentID string) (*mongo.Client, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), MgoTimeoutDialInfo)
 	defer cancel()
 
-	opts := options.Client().
-		ApplyURI(dsn).
+	opts, err := mongo_fix.ClientForDSN(dsn)
+	if err != nil {
+		panic(err)
```

Comment:
it is test, so we can `panic`

Reply:
okay, here we return error, so I think we can just propagate error.
but what about other cases, like in `func OpenTestMongoDB(tb testing.TB, dsn string) *mongo.Client {`
we don't have error in the API, do you think we should refactor API or we can panic (in tests only)

Reply:
we can call `require.NoError(tb, err)` in that places. so it will stop test with failed result

Reply:
yeah, good idea